README.md

```diff
@@ -16,14 +16,14 @@ The FireFly API for digital assets, data flows, and blockchain transactions make
 
 The best place to learn about FireFly is in the [documentation](https://hyperledger.github.io/firefly).
 
-There you will find our [Getting Started Guide](https://hyperledger.github.io/firefly/gettingstarted/),
+There you will find our [Getting Started Guide](https://hyperledger.github.io/firefly/latest/gettingstarted/),
 which will get you a running FireFly network of Supernodes on your local machine in a few minutes.
 
 Your development environment will come with:
 
 FireFly CLI                   |  FireFly Explorer UI                | FireFly Sandbox  |
 :----------------------------:|:-----------------------------------:|:----------------:|
-![](./images/firefly_cli.png) | ![](./images/firefly_explorer.png) | ![](./images/firefly_sandbox.png) |
+[![CLI](./images/firefly_cli.png)](https://hyperledger.github.io/firefly/latest/gettingstarted/firefly_cli/#install-the-firefly-cli) | [![UI](./images/firefly_explorer.png)](https://github.com/hyperledger/firefly-ui) | [![Sandbox](./images/firefly_sandbox.png)](https://hyperledger.github.io/firefly/latest/gettingstarted/sandbox/#use-the-sandbox) |
```

Comment:
The path to these images still seem incorrect

Reply:
@dwertent this doesn't really seem to work, a link on a link to an image

Reply:
<img width="540" alt="image" src="https://github.com/hyperledger/firefly/assets/30461857/509df7f5-52eb-4b0d-886b-27b765b28800">

Reply:
Maybe new syntax of `<img>` ? https://stackoverflow.com/questions/61071158/add-image-with-link-in-githubs-readme-md

Reply:
or no trailing `]`

Reply:
@EnriqueL8 Where did you get the screenshot from?
It looks like [it works](https://github.com/dwertent/firefly/tree/patch-1?tab=readme-ov-file#start-using-hyperledger-firefly):

![image](https://github.com/hyperledger/firefly/assets/64066841/5ea6db84-cc3e-41ec-aa0e-403d60768ac1)

Reply:
I got it from the diff in the PR https://github.com/hyperledger/firefly/pull/1516/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5